Github/service.js

```diff
@@ -33,14 +32,16 @@ module.exports.getRepos = recency => {
           console.log('Error:', res.statusMessage);
           reject(res.Error);
         } else {
-          const repos = JSON.parse(data);
-          repos.forEach(repo => {
-            if (today - new Date(repo.pushed_at) < recency) {
-              reposNames.push({
+          const reposNames = JSON.parse(data)
+            .filter(rep => {
+              return today - new Date(rep.pushed_at) < recency;
+            })
+            .map(repo => {
+              return {
                 name: repo.name,
-              });
-            }
-          });
+                language: repo.language,
```

Comment:
I think we should pass languages as an array since in the future we want to display multiple technologies. 

```suggestion
                languages: [repo.language],
```

Note that the repo url returns only top 1 language; github api provides additional url endpoint for getting all the languages for the repo `https://api.github.com/repos/{{organization}}/{{repo}}/languages`

Reply:
This is only a temporary solution since we still don't know if `technology` will be coming from labels or issue's textbox. An array of languages wouldn't work either since it wouldn't reflect the technologies for that specific issue but for the whole repo.